koji/flatten-el8/koji-ansible/roles/grobisplitter/files/rhel8-split.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+HOMEDIR=/root/koji-sync/podman/
+BINDIR=/usr/local/bin
+
+ARCHES="x86_64"
+DATE=$(date -Ih | sed 's/+.*//')
+
+DATEDIR=${HOMEDIR}/koji/${DATE}
+
+if [ -d ${DATEDIR} ]; then
+    echo "Directory already exists. Please remove or fix"
+    exit
+else
+mkdir -p ${DATEDIR}
```

Comment:
Nit: indenting

Reply:
I'd prefer not to deviate from their script more than necessary at this time.